src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -122,7 +124,7 @@ function validateExplicitKey(element, parentType) {
     '%s%s See https://fb.me/react-warning-keys for more information.%s',
     currentComponentErrorInfo,
     childOwner,
-    ReactComponentTreeHook.getCurrentStackAddendum(element)
+    getCurrentStackAddendum && getCurrentStackAddendum(element)
```

Comment:
Why do we need this check, here and in files below? This method is defined unconditionally, and AFAIK all the callers are also DEV-only files.